src/stealth/hiding_tricks.js

```diff
@@ -1,10 +1,16 @@
 /* istanbul ignore file */
 
 const hideWebDriver = () => {
-    const newProto = window.navigator.__proto__; // eslint-disable-line
-    delete newProto.webdriver;
-    // eslint-disable-next-line
-    navigator.__proto__ = newProto
+    Object.defineProperty(window, 'navigator', {
```

Comment:
Why redefine the whole navigator and not just the webdriver property?

Reply:
Because the property webdriver should not be there and honestly I don't know how the check is implemented. I guess that when you mocking the webdriver property you can still check if the property exists on the navigator object with something like `hasOwnProperty()` and that the check would catch us.